Anytype/Sources/PresentationLayer/TextEditor/AccessoryView/Mention/MentionViewController/MentionsViewController.swift

```diff
@@ -105,20 +105,12 @@ final class MentionsViewController: UITableViewController {
         if objectName.isEmpty {
             configuration.text = Loc.createNewObject
         } else {
-            let mutableAttributedString = NSMutableAttributedString(
-                string: Loc.createObject,
-                attributes: [.font: UIFont.uxTitle2Regular, .foregroundColor: UIColor.Text.primary]
-            )
-            let nameAttributedString = NSAttributedString(
-                string: objectName,
+            let string = Loc.createObject + " \"\(objectName)\""
```

Comment:
please use parameters for change for change word order in different languages

Reply:
@joe-pusya FYI.

Reply:
sure, fixed